Flick/Cells/ProfileSummaryTableViewCell.swift

```diff
@@ -106,12 +113,18 @@ class ProfileSummaryTableViewCell: UITableViewCell {
 
         userInfoView.snp.makeConstraints { make in
             make.top.equalTo(nameLabel.snp.bottom).offset(6)
+            make.bottom.equalTo(bioLabel.snp.top).offset(-8)
```

Comment:
Same question about top and bottom constraint + height?

Reply:
In general, this view has a lot of constraints hahah

Reply:
haha it is a lot of constraints.  I did it so the cell could auto-resize because bio could be multiple lines. The cell will resize appropriately to satisfy all the constraints so we don't have to set a strict height for the cell